content/TruenasApps/StableApps/InstallNextCloudMedia.md

```diff
@@ -15,151 +15,284 @@ keywords:
+### Application Name Settings
 
-   b. Select **Host Path (Path that already exists on the system)** in **Type**, then browse to and select the **userdata** dataset to populate the **Host Path** for the **Nextcloud User Data Storage** fields.
+{{< include file="/static/includes/apps/AppsWizardAppNameAndVersion.md" >}}
 
-   c. Scroll down to the **Nextcloud Postgres Data Storage** option.
-      Select **Host Path (Path that already exists on the system)** in **Type**, then browse to and select the **pgpdata** dataset to populate the **Host Path**.
+### Configuration Settings
+Nextcloud configuration settings include setting up credentials, ACT packages (previously referred to as the commands), the host IP and port, data directory path, upload limits, execution times, memory limits and cache memory consumption, adding a cron job with schedule, and adding additional environment variables.
 
-   d. Scroll down to **Nextcloud Postgres Backup Storage**, select **Host Path**, and then enter or browse to the path for the **pgbbackup** dataset.
-      When complete, the four datasets for Nextcloud are configured.
+If you have an existing Nextcloud account add the credentials for that account in the **Admin User** and **Admin Password** fields.
+If you do not have an existing account enter the name and password you want to use to create the Nextcloud login credentials.
 
-    {{< trueimage src="/images/SCALE/Apps/InstallNextcloudStorageConfigPostgressHostPaths.png" alt="Add Nextcloud Storage Volumes" id="Add Nextcloud Storage Volumes" >}}
+{{< expand "Adding APT Packages" "v" >}}
+Nextcloud has three APT package options:
+* **ffmpeg**
+* **smbclient**
+* **ocrmypdf**
 
-6. Accept the remaining setting defaults.
+{{< trueimage src="/images/SCALE/Apps/InstallNextcloudConfigAPTPackageOptions.png" alt="Configure Nextcloud APT Packages" id="Configure Nextcloud APT Packages" >}}
 
-7. Scroll up to review the configuration settings and fix any errors or click **Install** to begin the installation.
+You must add both the **ffmpeg** and **smbclient** packages to deploy this app.
+You can use **ocrmypdf** as well if needed, but you must also select the **Tesseract Language Code** to use. Options are **chi-sim** for Simplified Chinese or **eng** for English.
+Click **Add** to the right of **APT Packages** for each option you want or need to add.
+{{< /expand >}}
+To configure the host, enter the IP address for the TrueNAS system where you install the Nextcloud app and the web port number separated by a colon. For example, <i>12.123.12.3</i><b>:30027</b>.
+**30027** is the default port assigned to the TrueNAS Nextcloud app deployment. See [Network Configuration](#network-configuration) for more information on this port assignment and how to change it if so desired.
```

Comment:
Dont we have this section above?
Also there is a screenshot of nextcloud at login page, but then we go again over installation settings?